cmd/api/api.go

```diff
@@ -118,15 +118,19 @@ func NewService(cfg *config.ServerConfig) (*Service, error) {
 	if err != nil {
 		return nil, err
 	}
-	runtimeClients := make(map[common.Runtime]nodeapi.RuntimeApiLite)
+	runtimeClients := make(map[common.Runtime]storage.RuntimeClient)
 	if cfg.Node != nil {
 		apiRuntimes := []common.Runtime{common.RuntimeEmerald, common.RuntimeSapphire}
 		for _, runtime := range apiRuntimes {
 			client, err := source.NewRuntimeClient(ctx, cfg.Node, runtime)
 			if err != nil {
 				return nil, err
 			}
-			runtimeClients[runtime] = client
+			runtimeClients[runtime] = storage.RuntimeClient{
+				Runtime:           runtime,
+				Client:            client,
+				NativeTokenSymbol: cfg.Node.SDKNetwork().ParaTimes.All[string(runtime)].Denominations[sdkConfig.NativeDenominationKey].Symbol,
```

Comment:
could we save the whole ParaTimes.All[...] paratime structure here, then we'd have data about the decimals later too

Reply:
Good idea, updated!